garminexport/garminbackup.py

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env python
+"""Performs (incremental) backups of activities for a given Garmin Connect
+account.
+The activities are stored in a local directory on the user's computer.
+The backups are incremental, meaning that only activities that aren't already
+stored in the backup directory will be downloaded.
+"""
+import argparse
+from datetime import timedelta
+import getpass
+from garminexport.garminclient import GarminClient
+import garminexport.backup
+from garminexport.backup import export_formats
+from garminexport.retryer import (
+    Retryer, ExponentialBackoffDelayStrategy, MaxRetriesStopStrategy)
+import logging
+import os
+import re
+import sys
+import traceback
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)-15s [%(levelname)s] %(message)s")
+log = logging.getLogger(__name__)
+
+LOG_LEVELS = {
+    "DEBUG": logging.DEBUG,
+    "INFO": logging.INFO,
+    "WARNING": logging.WARNING,
+    "ERROR": logging.ERROR
+}
+"""Command-line (string-based) log-level mapping to logging module levels."""
+
+DEFAULT_MAX_RETRIES = 7
+"""The default maximum number of retries to make when fetching a single activity."""
+
+
+def parse_args():
```

Comment:
I would prefer to have this in a separate file: `garminexport/cli.py`.
Also, add a brief description of the function.

Reply:
Actually, also mention that the returned dict can be used directly with the `garminbackup()` function.

Reply:
done. cli.py is there